lms/djangoapps/api_manager/users/views.py

```diff
@@ -1012,6 +1009,13 @@ def get(self, request, user_id, course_id):
         if len(queryset):
             current_grade = queryset[0].grade
             proforma_grade = queryset[0].proforma_grade
+            progress_summary = json.loads(queryset[0].progress_summary)
+            grade_summary = json.loads(queryset[0].grade_summary)
+            grading_policy = json.loads(queryset[0].grading_policy)
+        else:
```

Comment:
The reason why I added the else statement is that I saw we initialized grades to a value of zero before the if statement, so I guessed that summaries and grading_policy should be dealt with in the same fashion.

My initial thought was to initialize the three as an `{}` but didn't know how apros would respond to that, since it expected summaries and grading policy to have certain key/value pairs (I believe it would result in KeyError exception or something similar). So, because of that, I've decided to add the else statement, thus providing proper initialization in case if condition returns false (e.g. database table has no records for any reason).

This is just a safety precaution, but if you think we can do without it or have a better alternative for this, please let me know.